build.sbt

```diff
@@ -0,0 +1,206 @@
+import com.amazonaws.auth.{EnvironmentVariableCredentialsProvider, InstanceProfileCredentialsProvider}
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+import com.typesafe.sbt.packager.docker.ExecCmd
+import sbt.Tests.SubProcess
+import sbtrelease.ReleaseStateTransformations._
+import scalariform.formatter.preferences.{AlignArguments, AlignParameters, AlignSingleLineCaseStatements, CompactControlReadability, DanglingCloseParenthesis, DoubleIndentClassDeclaration, FormatXml, FormattingPreferences, IndentSpaces, IndentWithTabs, MultilineScaladocCommentsStartOnFirstLine, PlaceScaladocAsterisksBeneathSecondAsterisk, Preserve, PreserveSpaceBeforeArguments, SpaceBeforeColon, SpaceInsideBrackets, SpaceInsideParentheses, SpacesAroundMultiImports, SpacesWithinPatternBinders}
+
+lazy val IntegrationTest = config("integration") extend Test
+lazy val formattingTestArg = Tests.Argument("-eDFG")
+
+// 0.1.15 has tons of false positives in async/await
+resolvers += Resolver.sonatypeRepo("snapshots")
+addCompilerPlugin("org.psywerx.hairyfotr" %% "linter" % "0.1-SNAPSHOT")
```

Comment:
The last commit on linter is the 0.1.15 release. So 0.1-SNAPSHOT should not be newer...